main.go

```diff
@@ -225,24 +225,26 @@ func main() {
 	)
 	checkNoErr(pauseResult.Err)
 
+	log.Printf("Bridge deployed and paused...")
+
 	/* --- USDCFlow TokenHandler Configuration --- */
 
 	// Add USDCFlow TokenHandler
-	usdcFlowAddr := o.Address("USDCFlow")
-	usdcFlowVaultIdentifier := buildUSDCFlowVaultIdentifier(usdcFlowAddr)
-	usdcFlowMinterIdentifier := buildUSDCFlowMinterIdentifier(usdcFlowAddr)
+	// usdcFlowAddr := o.Address("USDCFlow")
```

Comment:
Why is this commented out? Is this because we had to leave out the bridge related stuff in USDCFlow because it didn't migrate properly?

Reply:
Yeah, since USDCFlow won't be migrated with bridge dependencies, we won't be able to configure a `CadenceNativeTokenHandler` for in the process of deploying the bridge contracts. We'll still be able to configure it before activating the bridge, it will just need to be done outside of the context of this script.